app/components/SCForm.js

```diff
@@ -96,9 +96,12 @@ class SCForm extends Component {
   onChange(value) {
     const formInfo = this.props.navigation.state.params.form;
     const length = formInfo.fields.length;
-    let newState = {};
     let fieldValue, fieldKey, max, min;
     let fieldValidations = [];
+    let newState = ({
+      ...this.state,
+      field_key: fieldValue
+    });
```

Comment:
What is the purpose of this block of code? `field_key` and `fieldValue` are not defined at this point. also, the value of the form is stored in `this.state.value`.